datahub-graphql-core/src/main/resources/entity.graphql

```diff
@@ -4640,6 +4834,77 @@ input CreateTagInput {
   description: String
 }
 
+"""
+Input required to create/update a new ERModelRelation
+"""
+input ERModelRelationUpdateInput {
+    """
+    Details about the ERModelRelation
+    """
+    properties: ERModelRelationPropertiesInput
+    """
+    Update to editable properties
+    """
+    editableProperties: ERModelRelationEditablePropertiesUpdate
```

Comment:
we can get rid of this since we can simplify things and get rid of the editable aspect

Reply:
Did not remove because we use it
EditableERModelRelationProperties stores editable changes made to ermodelrelation properties. This separates changes made from ingestion pipelines and edits in the UI to avoid accidental overwrites of user-provided data by ingestion pipelines

Reply:
okay that's cool!